### PR TITLE
Bug with false alarm on weekend

### PR DESCRIPTION
**Pull request by miiila**
Fix of https://github.com/apiaryio/pagerduty-overlap-checker/issues/27. One more bug discovered (similar situation, opposite direction), fixed in https://github.com/apiaryio/pagerduty-overlap-checker/commit/7fb1079852ebcd4a64e01fcc9ac0d73730f294a7.

For better readability and further development, date function were refactored for moment.js and message sent to the slack was updated with times of overlaps, not the times of the shifts.

After merge:
- [ ] deploy the new version to lambda

### CODE REVIEW

**Comment by miiila** [comment]
[Overlap checker bug (Spike)](https://trello.com/c/2mFN26ov/711-overlap-checker-bug-spike)

File: src/pagerduty.js [hunk 1]
```diff
@@ -8,15 +9,13 @@ const pdApi = require('./pagerduty-api');
...
-    const timeNow = new Date();
-    const timeUntil = new Date(timeNow.getTime() + (nconf.get('WEEKS_TO_CHECK') * week));
+    const timeUntil = moment.utc().utc().add(nconf.get('WEEKS_TO_CHECK'), 'w');
 
     const scheduleOpts = {
       qs: {
         'schedule_ids[]': id,
         until: timeUntil.toISOString(),
-        since: timeNow.toISOString(),
+        since: moment.utc().utc().toISOString(),
```
Comment (abtris) [inline src/pagerduty.js]: 2 times utc() function?

File: src/pagerduty.js [hunk 2]
```diff
@@ -8,15 +9,13 @@ const pdApi = require('./pagerduty-api');
 // Get schedule for ID and 2 weeks
 function getSchedule(id, cb) {
   if (nconf.get('WEEKS_TO_CHECK') > 0) {
-    const week = 7 * 86400 * 1000;
-    const timeNow = new Date();
-    const timeUntil = new Date(timeNow.getTime() + (nconf.get('WEEKS_TO_CHECK') * week));
+    const timeUntil = moment.utc().utc().add(nconf.get('WEEKS_TO_CHECK'), 'w');
```
Comment (abtris) [inline src/pagerduty.js]: 2 times utc() function?

File: src/pagerduty.js (hunk 1 above)
Comment (miiila) [reply src/pagerduty.js]: Good catch - `replace all` went wild ;-)

**Pull request merged**